### PR TITLE
mayhem integration

### PR DESCRIPTION
**Pull request by evanrichter**

### CODE REVIEW

File: relay/sources/relayd/fuzz/Cargo.toml [hunk 1]
```diff
@@ -7,22 +7,17 @@ publish = false
 [package.metadata]
```
Comment (xansec) [inline relay/sources/relayd/fuzz/Cargo.toml:7]: Can you tell me why these changes were needed?

**Review by xansec — CHANGES_REQUESTED** [review]
Is this fuzzing rudder or relayd (https://github.com/reyk/relayd)? If the latter, I wouldn't consider this a rudder integration as much as a relayd integration. Please make the requested changes.

File: relay/sources/relayd/fuzz/Cargo.toml (hunk 1 above)
Comment (evanrichter) [reply relay/sources/relayd/fuzz/Cargo.toml:7]: this is all just updating an old fuzz harness that didn't compile due to name changes and API changes. the Cargo.toml file above this one declares the package name to "rudder-relayd" so that had to be updated. The libfuzzer-sys dep and harness style also were the old style, so I simplified it

**Comment by evanrichter** [comment]
I haven't seen that repo! but I cloned it and did a folder comparison against this repo's _relay/sources/relayd_ directory to be sure. It's definitely a new project since the other one is completely C based, and this one is 100% rust

**Review by xansec — APPROVED** [review]
This makes sense to me. Merging. LGTM!
